### ISSUE TITLE
PIA Next gen portfowarding

### ISSUE THREAD

**Issue by sbeckeriv** [binhex/arch-int-openvpn#14]
Dearest maintainer,

I have not been able to connect to the port forwarding IP for a few days. I have had others test it outside my network without any success either. PIA Support said "Please know that we are fully aware of the issue and we are doing our best to resolve this the soonest as possible we can."  

There are a number of reddit threads about this issue and about the next gen port forwarding [1] [2]. I am honestly out of my depth with this kind of thing. I have aggregated some links to in progress work and scripts people have written for the next gen port forwarding [3] [4] [5]. 

There is no real action here. I just wanted to bring it to your attention. Please feel free to close it or leave it open to track a migration to next gen port forwarding.

Thank you for all of your time and work on this project

Dictated but not reviewed,
Becker

[1] https://www.reddit.com/r/PrivateInternetAccess/comments/iqt5aq/screw_you_pia/g4u4e1m/?utm_source=reddit&utm_medium=web2x&context=3
[2] https://www.reddit.com/r/PrivateInternetAccess/comments/i6qqu0/pia_portforward_request_ip_is_dead/
[3] https://github.com/thrnz/docker-wireguard-pia/blob/master/extra/pf.sh
[4] https://gist.github.com/triffid/da48f3c99f1ff334571ae49be80d591b#file-pia-portforward-sh
[5] https://github.com/qdm12/gluetun/issues/236#issuecomment-691291388

**Comment by binhex** [comment]
thanks for the links, here is my response to the ongoing issue with pia:-
https://github.com/binhex/arch-delugevpn/issues/194#issuecomment-691044876

**Comment by 0-MegaMind-0** [comment]
when I made [this ](https://www.reddit.com/r/PrivateInternetAccess/comments/i6qqu0/pia_portforward_request_ip_is_dead/)thread  PIA did deny that they had issues with the API as it was working for them. But I did forget to update that thread as I got busy, I can confirm that the Romanian server mostly works fine at this time but API is/was completely dead when connected to the Spain Server.(test around the few old servers until you get one that works reliably enough until this is fixed)

u/Threarah did provide his docker container for me to test but as I was busy with my private project and never got around to testing it. But since PIA has now provided [official beta support](https://www.privateinternetaccess.com/helpdesk/news/posts/introducing-next-generation-port-forwarding) for portforwarding and wireguard without PIA client was going to hack together a test docker with rtorrent to see if it works reliably enough for wireguard.

@binhex are you planning on adding wiregaurd support too?
